server/main.py

```diff
@@ -224,9 +217,17 @@ async def index(request):
             f'gsutil cp {METADATA_PREFIX}.json {quote(output_dir)}/metadata.json'
         )
 
+        job.command(
```

Comment:
Should this be dependent on the checks above? (E.g. you wouldn't want to `chmod +x /bin/bash`.)

Reply:
That check is performed in the `cli`, but it makes sense to repeat these checks here.